src/redux/reducers/genes/genesExpressionLoaded.js

```diff
@@ -55,7 +45,10 @@ const genesExpressionLoaded = (state, action) => {
           data: genes,
         },
       },
-      loading: loadingStatus,
+      full: {
```

Comment:
what is the idea behind adding `full`?

Reply:
yep, it's the full expression, used in embedding for example